lib/Makefile.am

```diff
@@ -258,6 +259,7 @@ if LIBMESH_ENABLED
   ../include/ibamr/CIBFEMethod.h \
   ../include/ibamr/IBFECentroidPostProcessor.h \
   ../include/ibamr/IBFEMethod.h \
+  ../include/ibamr/IBFEInstrumentPanel.h \
```

Comment:
IBFEInstrumentPanel.h should go above IBFEMethod.h (try to keep everything in alphabetical order).